javascripts/govuk/show-hide-content.js

```diff
@@ -0,0 +1,150 @@
+(function (global) {
+  'use strict';
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+
+  function ShowHideContent() {
+    var self = this;
+
+    // Radio and Checkbox selectors
+    var selectors = {
+      namespace: 'ShowHideContent',
+      radio: '.block-label input[type="radio"]',
+      checkbox: '.block-label input[type="checkbox"]'
+    };
+
+    // Escape name attribute for use in DOM selector
+    function escapeElementName(str) {
+      var result = str.replace('[', '\\[').replace(']', '\\]');
+      return(result);
+    }
+
+    // Adds ARIA attributes to control + associated content
+    function initToggledContent() {
+      var $control = $(this);
+      var $content = getToggledContent($control);
+
+      // Set aria-controls and defaults
+      if ($content.length) {
+        $control.attr('aria-controls', $content.attr('id'));
+        $control.attr('aria-expanded', 'false');
+        $content.attr('aria-hidden', 'true');
+      }
+    }
+
+    // Return toggled content for control
+    function getToggledContent($control) {
```

Comment:
@tombye I've just improved `getToggledContent` instead. I'd only want one method to have ownership of "get the content" so didn't want to have another version in the init methods.

The new version defaults to checking `aria-controls` for the ID.